src/RA_Dlg_MemBookmark.cpp

```diff
@@ -651,17 +648,21 @@ void Dlg_MemBookmark::WriteFrozenValue(const MemBookmark & Bookmark)
             break;
     }
 
-    char buffer[32];
-    sprintf_s(buffer, sizeof(buffer), "%0*x", width, Bookmark.Value());
+    // spectre stuff, needs to be a string
+    std::string buffer;
+    buffer.reserve(32);
+    sprintf_s(buffer.data(), 32, "%0*x", width, Bookmark.Value());
+    buffer = buffer.data();
```

Comment:
This is required before using the ranged for. The string is in memory but hasn't been saved to the object yet.